lib/classes/eik-config.js

```diff
@@ -82,7 +171,7 @@ module.exports = class EikConfig {
                 return [[pathname, filePaths[0], pathname, file]];
             }
             return filePaths.map((filePath) => [
-                path.join(pathname, path.basename(filePath)),
+                path.join(pathname, filePath),
```

Comment:
Do you want to change this function if it is being deprecated in favour of `mappings`? Should it be marked as deprecated?

Reply:
Yea, probably should do something here true. I could also just delete them now that I have things working and this release is super breaking anyway maybe

Reply:
I guess it is only breaking if you use the new function? Could keep the old ones around until everything has been migrated. Though not sure if that helps. Might be nicer to see breaking tests rather than have everything working.

Reply:
yeah, kinda tempted to go hard breaking here and rip off the bandage.